.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,15 @@
+## Summary of changes
+
+- _Code/resources that have been added, updated or removed. Please provide rationale for changes whenever applicable_
+
+## How to Test
+
+- _Link to staging deployment and credentials whenever applicable_
```

Comment:
I was thinking I could leave this in case there's a testing environment a contributor may want to showcase. Better remove it?